.github/workflows/libjs-test262.yml

```diff
@@ -133,3 +133,39 @@ jobs:
         continue-on-error: true
         working-directory: libjs-test262
         run: ./per_file_result_diff.py -o per-file-bytecode-master.json -n ../libjs-website/test262/data/per-file-bytecode-master.json
+
+      - name: Build js
+        working-directory: libjs-test262/Build
+        run: |
+          ninja js
+
+      - name: Create js package
+        working-directory: libjs-test262/Build
+        run: |
+          mkdir -p pkg/bin pkg/lib
+          cp --preserve _deps/lagom-build/js pkg/bin
+
+          for lib in $(find _deps/lagom-build -type f -name liblagom-*.so.0.0.0) ; do
+            dst=$(basename -s ".0.0" ${lib})
+            cp --preserve ${lib} pkg/lib/${dst}
+          done
+
+          cd pkg
+
+          echo "#!/bin/sh" > js
+          echo "DIRNAME=\"\$(dirname \$(readlink -f \$0))\"" >> js
+          echo "export LD_LIBRARY_PATH=\"\${DIRNAME}/lib\"" >> js
+          echo "exec \"\${DIRNAME}/bin/js\" \"\$@\"" >> js
+          chmod +x js
+
+          strip bin/js
+          strip lib/*
+
+          tar -cvf serenity_js.tar js bin lib
+          gzip serenity_js.tar
+
+      - name: Upload js package
+        uses: actions/upload-artifact@v2
+        with:
+          name: serenity-js
+          path: libjs-test262/Build/pkg/serenity_js.tar.gz
```

Comment:
Can we ensure we only keep these for a limited time? (Since we're only interested in latest anyways)

Reply:
@linusg FYI

Reply:
Our retention time is 90 days at the moment.

Reply:
That should be reduced significantly. (As this is the only artifact we have in this repo, and we are going to be generating these on every push to master)

Reply:
Hit me with a number and I'll put it in the box

Reply:
I mean the minimum (1 day) would probably work as long as we ensure we always get at least 1 commit in a day :)

Reply:
But we might want to check older builds for some reason, so a week would probably do.

Reply:
All right, 7 days is the new setting 👍🏻

Reply:
You can specify it per action with: `retention-days: 5` (in the with)